packages/sources/tradermade-test/src/endpoint/forex.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { BatchEndpointTypes, buildBatchedRequestBody, constructEntry } from '../price-utils'
+
+export const inputParameters = {
+  base: {
+    aliases: ['from', 'coin', 'symbol'],
+    required: true,
+    type: 'string',
+    description: 'The symbol of symbols of the currency to query',
+  },
+  quote: {
+    aliases: ['to', 'market', 'convert'],
+    required: true,
+    type: 'string',
+    description: 'The symbol of the currency to convert to',
+  },
+  transport: {
+    description: 'which transport to route to',
+    required: false,
+    type: 'string',
+  },
```

Comment:
We can also add `ws` and `rest` as options for this input param to enforce the correct transport names

Reply:
Currently typescript isn't allowing to use options (throws type errors). I think we need to change [this](https://github.com/smartcontractkit/ea-framework-js/blob/main/src/validation/input-params.ts#L11) to be readonly as other array values

Reply:
I think there's also some specific typing on the transport input param for routing transports. We can skip on this then